<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterEdit</class>
 <widget class="QDialog" name="RegisterEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>307</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>288</width>
    <height>74</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="analogGain">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="expo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Analog gain value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure value, ms</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Apply">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="digitalGain">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Digital gain value</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="AG">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>AG</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="AE">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>AE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> value</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>FPGA register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Value</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Value</string>
   </property>
  </widget>
  <widget class="QComboBox" name="addr">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>52</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>52</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="val">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>52</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>235</y>
     <width>283</width>
     <height>62</height>
    </rect>
   </property>
   <property name="title">
    <string>Actual values</string>
   </property>
   <widget class="QLabel" name="Lines">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Tacts">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ValGd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ValGa">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>AnalogG:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>DigitalG:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Lines:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Tacts:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Video Sensor Settings</string>
   </property>
  </widget>
  <zorder>groupBox_4</zorder>
  <zorder>analogGain</zorder>
  <zorder>expo</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>Apply</zorder>
  <zorder>digitalGain</zorder>
  <zorder>label_4</zorder>
  <zorder>AG</zorder>
  <zorder>AE</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>getButton</zorder>
  <zorder>setButton</zorder>
  <zorder>addr</zorder>
  <zorder>label</zorder>
  <zorder>val</zorder>
  <zorder>groupBox_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
